security.html

```diff
@@ -46,11 +46,12 @@
     </section>
     <section>
       <div class="container bg-shade">
-        <h1 class="section-heading">Repository Name</h1>
-	      <h2 class="section-heading">Report Name</h2>
+        <h1 class="section-heading">simple-event-sender</h1>
+	      <h2 class="section-heading">Several open security alerts</h2>
           <p class="section-paragraph">
 	          <ul>
-	            <li>details</li>
+	            <li>The simple-simple-event sender has several <a href="https://github.com/eiffel-community/simple-event-sender/security/dependabot">high severity security alerts</a>. All users should ensure dependencies are updated.</li>
```

Comment:
I get a 404 error for https://github.com/eiffel-community/simple-event-sender/security/dependabot.

Reply:
@magnusbaeck well spotted, seems like I have the rights to see that page so I didn't notice. How do we handle that? Should i add a short explanation on the security page?

Reply:
Even if the link in the PR worked for everyone I'm not sure it be a good fit for this page since it lists _current_ problems (right?) while this page also should cover historical vulnerabilities? Unless we build something very complicated I think this list needs to be maintained by hand.

Reply:
I get the 404 as well. Are the alerts still there? I seem to recall I've had access to the alerts page. But perhaps that was through a screen share? There are a also 2 open issued filed by dependabot on the repository in question. I can't remember if these were related.

Reply:
Probably only admin or simlar for the repo that can see it. Ok if I add a short description for now so that we get it published? @fredjn

Reply:
From https://github.com/eiffel-community/simple-event-sender/settings/security_analysis

>Dependabot and secret scanning alerts are only visible to people and teams that are given access by admins.

I don't think we can link to that page. So instead we need to change this to advise people to update to a certain version of simple-event-sender.

Reply:
@k-hallen-ericsson , could you make the necessary updates to this PR?